src/NzbDrone.Core/Indexers/TorrentRssIndexer/TorrentRssParserFactory.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using NLog;
+
+namespace NzbDrone.Core.Indexers.TorrentRssIndexer
+{
+    public class TorrentRssParserFactory : ITorrentRssParserFactory
+    {
+        private static readonly ITorrentRssParserSettingsCache SettingsCache = new TorrentRssParserSettingsCache();
+
+        protected readonly Logger _logger = LogManager.GetCurrentClassLogger();
```

Comment:
Add constructor and init via that constructor, let tinyioc instantiate the class.